client/components/risk-level/index.js

```diff
@@ -4,13 +4,14 @@
  * Internal dependencies
  */
 import riskMappings from './strings';
```

Comment:
Can we turn this file into TypeScript, without touching it's consumers?

Reply:
I have attempted this here 
7983cc8

Requesting advance forgiveness @kalessil  :)